test/Microsoft.DotNet.ToolPackage.Tests/LockFileMatcherTests.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using FluentAssertions;
+using Microsoft.DotNet.Tools.Test.Utilities;
+using Microsoft.Extensions.EnvironmentAbstractions;
+using Microsoft.DotNet.Cli;
+using Microsoft.DotNet.Tools.Install.Tool;
+using Xunit;
+
+namespace Microsoft.DotNet.ToolPackage.Tests
+{
+    public class LockFileMatcherTests : TestBase
+    {
+
+        [Fact]
+        public void MatchesEntryPointTests()
```

Comment:
I'd suggest making this a `Theory` instead of testing multiple things in one `Fact`.  As it stands, if the first assertion fails, you'll get no information about the subsequent ones.

Reply:
Fixed